tests/core/core.spec.ts

```diff
@@ -45,16 +45,42 @@ describe("@dataform/core", () => {
       expect(t.postOps).deep.equals(["post_op"]);
 
       const t2 = session
+        .publish("schema2.example", {
+          type: "table",
+          dependencies: ["schema1.example"],
```

Comment:
sorry for not catching this earlier: i don't think we should let users do this; `dependencies` should be a `Resolvable[]`, and thus elements should either be a string not containing `.`, or full `Resolvable` objects. (to clarify, we should make the `dependencies` method accept `dependencies: Resolvable | Resolvable[]`)

Reply:
Done. Thx for the catch.